packages/data-explorer/src/components/DataView/EntityCard.vue

```diff
@@ -0,0 +1,36 @@
+<template>
```

Comment:
Do we keep the use of the term entity ? , maybe ExplorerCard or something more ui centric ?